<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Get GWAS Catalog associations — get_associations • gwasrapidd</title>

<!-- favicons -->
<link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png" />
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png" />

<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>



  <link href="../extra.css" rel="stylesheet">
  

<meta property="og:title" content="Get GWAS Catalog associations — get_associations" />
<meta property="og:description" content="Retrieves associations via the NHGRI-EBI GWAS Catalog REST API. The REST
API is queried multiple times with the criteria passed as arguments (see
below). By default all associations that match the criteria supplied in the
arguments are retrieved: this corresponds to the default option
set_operation set to 'union'. If you rather have only the
associations that match simultaneously all criteria provided, then set
set_operation to 'intersection'." />
<meta property="og:image" content="https://rmagno.eu/gwasrapidd/logo.svg" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">gwasrapidd</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.99.11</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../articles/gwasrapidd.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/bmi_variants.html">Variants associated with Body Mass Index (BMI)</a>
    </li>
    <li>
      <a href="../articles/faq.html">Frequently asked questions</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/ramiromagno/gwasrapidd/">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Get GWAS Catalog associations</h1>
    <small class="dont-index">Source: <a href='https://github.com/ramiromagno/gwasrapidd/blob/master/R/get_associations.R'><code>R/get_associations.R</code></a></small>
    <div class="hidden name"><code>get_associations.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Retrieves associations via the NHGRI-EBI GWAS Catalog REST API. The REST
API is queried multiple times with the criteria passed as arguments (see
below). By default all associations that match the criteria supplied in the
arguments are retrieved: this corresponds to the default option
<code>set_operation</code> set to <code>'union'</code>. If you rather have only the
associations that match simultaneously all criteria provided, then set
<code>set_operation</code> to <code>'intersection'</code>.</p>
    </div>

    <pre class="usage"><span class='fu'>get_associations</span><span class='op'>(</span>
  study_id <span class='op'>=</span> <span class='cn'>NULL</span>,
  association_id <span class='op'>=</span> <span class='cn'>NULL</span>,
  variant_id <span class='op'>=</span> <span class='cn'>NULL</span>,
  efo_id <span class='op'>=</span> <span class='cn'>NULL</span>,
  pubmed_id <span class='op'>=</span> <span class='cn'>NULL</span>,
  efo_trait <span class='op'>=</span> <span class='cn'>NULL</span>,
  set_operation <span class='op'>=</span> <span class='st'>"union"</span>,
  interactive <span class='op'>=</span> <span class='cn'>TRUE</span>,
  verbose <span class='op'>=</span> <span class='cn'>FALSE</span>,
  warnings <span class='op'>=</span> <span class='cn'>TRUE</span>
<span class='op'>)</span></pre>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>study_id</th>
      <td><p>A <code>character</code> vector of GWAS Catalog study accession
identifiers.</p></td>
    </tr>
    <tr>
      <th>association_id</th>
      <td><p>A <code>character</code> vector of GWAS Catalog association
identifiers.</p></td>
    </tr>
    <tr>
      <th>variant_id</th>
      <td><p>A <code>character</code> vector of GWAS Catalog variant
identifiers.</p></td>
    </tr>
    <tr>
      <th>efo_id</th>
      <td><p>A character vector of <a href='https://www.ebi.ac.uk/efo/'>EFO</a>
identifiers.</p></td>
    </tr>
    <tr>
      <th>pubmed_id</th>
      <td><p>An <code>integer</code> vector of
<a href='https://en.wikipedia.org/wiki/PubMed'>PubMed</a> identifiers.</p></td>
    </tr>
    <tr>
      <th>efo_trait</th>
      <td><p>A <code>character</code> vector of
<a href='https://www.ebi.ac.uk/efo/'>EFO</a> trait descriptions, e.g.,
<code>'uric acid measurement'</code>.</p></td>
    </tr>
    <tr>
      <th>set_operation</th>
      <td><p>Either <code>'union'</code> or <code>'intersection'</code>. This
tells how associations retrieved by different criteria  should be combined:
<code>'union'</code> binds together all results removing duplicates and
<code>'intersection'</code> only keeps same associations found with different
criteria.</p></td>
    </tr>
    <tr>
      <th>interactive</th>
      <td><p>A logical. If all associations are requested, whether to ask
interactively if we really want to proceed.</p></td>
    </tr>
    <tr>
      <th>verbose</th>
      <td><p>A <code>logical</code> indicating whether the function should be
verbose about the different queries or not.</p></td>
    </tr>
    <tr>
      <th>warnings</th>
      <td><p>A <code>logical</code> indicating whether to print warnings, if any.</p></td>
    </tr>
    </table>

    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>An <a href='associations-class.html'>associations</a> object.</p>
    <h2 class="hasAnchor" id="details"><a class="anchor" href="#details"></a>Details</h2>

    <p>Please note that all search criteria are vectorised, thus allowing for batch
mode search, e.g., one can search by multiple variant identifiers at once by
passing a vector of identifiers to <code>variant_id</code>.</p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'><span class='co'># Get an association by study identifier</span>
<span class='fu'>get_associations</span><span class='op'>(</span>study_id <span class='op'>=</span> <span class='st'>'GCST001085'</span><span class='op'>)</span>
</div><div class='output co'>#&gt; An object of class "associations"
#&gt; Slot "associations":
#&gt; <span style='color: #555555;'># A tibble: 9 x 17</span><span>
#&gt;   association_id   pvalue pvalue_descript… pvalue_mantissa pvalue_exponent
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>                      </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span>           </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span>
#&gt; </span><span style='color: #555555;'>1</span><span> 24299694       1.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>-10</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             1             -</span><span style='color: #BB0000;'>10</span><span>
#&gt; </span><span style='color: #555555;'>2</span><span> 24299702       5.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>-13</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             5             -</span><span style='color: #BB0000;'>13</span><span>
#&gt; </span><span style='color: #555555;'>3</span><span> 24299710       3.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>-10</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             3             -</span><span style='color: #BB0000;'>10</span><span>
#&gt; </span><span style='color: #555555;'>4</span><span> 24299718       3.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>-16</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             3             -</span><span style='color: #BB0000;'>16</span><span>
#&gt; </span><span style='color: #555555;'>5</span><span> 24299726       2.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>-44</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             2             -</span><span style='color: #BB0000;'>44</span><span>
#&gt; </span><span style='color: #555555;'>6</span><span> 24299734       1.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>- 7</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             1              -</span><span style='color: #BB0000;'>7</span><span>
#&gt; </span><span style='color: #555555;'>7</span><span> 24299743       7.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>- 9</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             7              -</span><span style='color: #BB0000;'>9</span><span>
#&gt; </span><span style='color: #555555;'>8</span><span> 24299751       2.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>- 7</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             2              -</span><span style='color: #BB0000;'>7</span><span>
#&gt; </span><span style='color: #555555;'>9</span><span> 24299759       2.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>- 9</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             2              -</span><span style='color: #BB0000;'>9</span><span>
#&gt; </span><span style='color: #555555;'># … with 12 more variables: multiple_snp_haplotype </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   snp_interaction </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span style='color: #555555;'>, snp_type </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>, standard_error </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>, range </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   or_per_copy_number </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>, beta_number </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>, beta_unit </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   beta_direction </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>, beta_description </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>, last_mapping_date </span><span style='color: #555555;font-style: italic;'>&lt;dttm&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   last_update_date </span><span style='color: #555555;font-style: italic;'>&lt;dttm&gt;</span><span>
#&gt; 
#&gt; Slot "loci":
#&gt; </span><span style='color: #555555;'># A tibble: 18 x 4</span><span>
#&gt;    association_id locus_id haplotype_snp_count description          
#&gt;    </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span>               </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>                
#&gt; </span><span style='color: #555555;'> 1</span><span> 24299694              1                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'> 2</span><span> 24299694              2                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'> 3</span><span> 24299702              1                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'> 4</span><span> 24299702              2                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'> 5</span><span> 24299710              1                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'> 6</span><span> 24299710              2                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'> 7</span><span> 24299718              1                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'> 8</span><span> 24299718              2                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'> 9</span><span> 24299726              1                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'>10</span><span> 24299726              2                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'>11</span><span> 24299734              1                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'>12</span><span> 24299734              2                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'>13</span><span> 24299743              1                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'>14</span><span> 24299743              2                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'>15</span><span> 24299751              1                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'>16</span><span> 24299751              2                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'>17</span><span> 24299759              1                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'>18</span><span> 24299759              2                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; 
#&gt; Slot "risk_alleles":
#&gt; </span><span style='color: #555555;'># A tibble: 18 x 7</span><span>
#&gt;    association_id locus_id variant_id risk_allele risk_frequency genome_wide
#&gt;    </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>      </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>                </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span>      
#&gt; </span><span style='color: #555555;'> 1</span><span> 24299694              1 rs13420028 A                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'> 2</span><span> 24299694              2 rs10188442 A                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'> 3</span><span> 24299702              1 rs7735940  A                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'> 4</span><span> 24299702              2 rs12522034 C                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'> 5</span><span> 24299710              1 rs3798440  A                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'> 6</span><span> 24299710              2 rs9350602  C                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'> 7</span><span> 24299718              1 rs2469997  G                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'> 8</span><span> 24299718              2 rs6469823  C                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'> 9</span><span> 24299726              1 rs7827545  C                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'>10</span><span> 24299726              2 rs1372662  G                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'>11</span><span> 24299734              1 rs7960483  T                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'>12</span><span> 24299734              2 rs10785581 C                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'>13</span><span> 24299743              1 rs200752   T                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'>14</span><span> 24299743              2 rs200759   A                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'>15</span><span> 24299751              1 rs6452524  G                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'>16</span><span> 24299751              2 rs6887846  A                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'>17</span><span> 24299759              1 rs10496288 G                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'>18</span><span> 24299759              2 rs10496289 C                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'># … with 1 more variable: limited_list </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span>
#&gt; 
#&gt; Slot "genes":
#&gt; </span><span style='color: #555555;'># A tibble: 18 x 3</span><span>
#&gt;    association_id locus_id gene_name 
#&gt;    </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>     
#&gt; </span><span style='color: #555555;'> 1</span><span> 24299694              1 GPR39     
#&gt; </span><span style='color: #555555;'> 2</span><span> 24299694              2 GPR39     
#&gt; </span><span style='color: #555555;'> 3</span><span> 24299702              1 RANBP3L   
#&gt; </span><span style='color: #555555;'> 4</span><span> 24299702              2 RANBP3L   
#&gt; </span><span style='color: #555555;'> 5</span><span> 24299710              1 MYO6      
#&gt; </span><span style='color: #555555;'> 6</span><span> 24299710              2 MYO6      
#&gt; </span><span style='color: #555555;'> 7</span><span> 24299718              1 NOV       
#&gt; </span><span style='color: #555555;'> 8</span><span> 24299718              2 NOV       
#&gt; </span><span style='color: #555555;'> 9</span><span> 24299726              1 ZFAT      
#&gt; </span><span style='color: #555555;'>10</span><span> 24299726              2 ZFAT      
#&gt; </span><span style='color: #555555;'>11</span><span> 24299734              1 AN06      
#&gt; </span><span style='color: #555555;'>12</span><span> 24299734              2 AN06      
#&gt; </span><span style='color: #555555;'>13</span><span> 24299743              1 MACROD2   
#&gt; </span><span style='color: #555555;'>14</span><span> 24299743              2 MACROD2   
#&gt; </span><span style='color: #555555;'>15</span><span> 24299751              1 XRCC4     
#&gt; </span><span style='color: #555555;'>16</span><span> 24299751              2 XRCC4     
#&gt; </span><span style='color: #555555;'>17</span><span> 24299759              1 intergenic
#&gt; </span><span style='color: #555555;'>18</span><span> 24299759              2 intergenic
#&gt; 
#&gt; Slot "ensembl_ids":
#&gt; </span><span style='color: #555555;'># A tibble: 18 x 4</span><span>
#&gt;    association_id locus_id gene_name  ensembl_id     
#&gt;    </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>      </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>          
#&gt; </span><span style='color: #555555;'> 1</span><span> 24299694              1 GPR39      ENSG00000183840
#&gt; </span><span style='color: #555555;'> 2</span><span> 24299694              2 GPR39      ENSG00000183840
#&gt; </span><span style='color: #555555;'> 3</span><span> 24299702              1 RANBP3L    ENSG00000164188
#&gt; </span><span style='color: #555555;'> 4</span><span> 24299702              2 RANBP3L    ENSG00000164188
#&gt; </span><span style='color: #555555;'> 5</span><span> 24299710              1 MYO6       ENSG00000196586
#&gt; </span><span style='color: #555555;'> 6</span><span> 24299710              2 MYO6       ENSG00000196586
#&gt; </span><span style='color: #555555;'> 7</span><span> 24299718              1 NOV        </span><span style='color: #BB0000;'>NA</span><span>             
#&gt; </span><span style='color: #555555;'> 8</span><span> 24299718              2 NOV        </span><span style='color: #BB0000;'>NA</span><span>             
#&gt; </span><span style='color: #555555;'> 9</span><span> 24299726              1 ZFAT       ENSG00000066827
#&gt; </span><span style='color: #555555;'>10</span><span> 24299726              2 ZFAT       ENSG00000066827
#&gt; </span><span style='color: #555555;'>11</span><span> 24299734              1 AN06       </span><span style='color: #BB0000;'>NA</span><span>             
#&gt; </span><span style='color: #555555;'>12</span><span> 24299734              2 AN06       </span><span style='color: #BB0000;'>NA</span><span>             
#&gt; </span><span style='color: #555555;'>13</span><span> 24299743              1 MACROD2    ENSG00000172264
#&gt; </span><span style='color: #555555;'>14</span><span> 24299743              2 MACROD2    ENSG00000172264
#&gt; </span><span style='color: #555555;'>15</span><span> 24299751              1 XRCC4      ENSG00000152422
#&gt; </span><span style='color: #555555;'>16</span><span> 24299751              2 XRCC4      ENSG00000152422
#&gt; </span><span style='color: #555555;'>17</span><span> 24299759              1 intergenic </span><span style='color: #BB0000;'>NA</span><span>             
#&gt; </span><span style='color: #555555;'>18</span><span> 24299759              2 intergenic </span><span style='color: #BB0000;'>NA</span><span>             
#&gt; 
#&gt; Slot "entrez_ids":
#&gt; </span><span style='color: #555555;'># A tibble: 18 x 4</span><span>
#&gt;    association_id locus_id gene_name  entrez_id
#&gt;    </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>      </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>    
#&gt; </span><span style='color: #555555;'> 1</span><span> 24299694              1 GPR39      2863     
#&gt; </span><span style='color: #555555;'> 2</span><span> 24299694              2 GPR39      2863     
#&gt; </span><span style='color: #555555;'> 3</span><span> 24299702              1 RANBP3L    202151   
#&gt; </span><span style='color: #555555;'> 4</span><span> 24299702              2 RANBP3L    202151   
#&gt; </span><span style='color: #555555;'> 5</span><span> 24299710              1 MYO6       4646     
#&gt; </span><span style='color: #555555;'> 6</span><span> 24299710              2 MYO6       4646     
#&gt; </span><span style='color: #555555;'> 7</span><span> 24299718              1 NOV        4856     
#&gt; </span><span style='color: #555555;'> 8</span><span> 24299718              2 NOV        4856     
#&gt; </span><span style='color: #555555;'> 9</span><span> 24299726              1 ZFAT       57623    
#&gt; </span><span style='color: #555555;'>10</span><span> 24299726              2 ZFAT       57623    
#&gt; </span><span style='color: #555555;'>11</span><span> 24299734              1 AN06       </span><span style='color: #BB0000;'>NA</span><span>       
#&gt; </span><span style='color: #555555;'>12</span><span> 24299734              2 AN06       </span><span style='color: #BB0000;'>NA</span><span>       
#&gt; </span><span style='color: #555555;'>13</span><span> 24299743              1 MACROD2    140733   
#&gt; </span><span style='color: #555555;'>14</span><span> 24299743              2 MACROD2    140733   
#&gt; </span><span style='color: #555555;'>15</span><span> 24299751              1 XRCC4      7518     
#&gt; </span><span style='color: #555555;'>16</span><span> 24299751              2 XRCC4      7518     
#&gt; </span><span style='color: #555555;'>17</span><span> 24299759              1 intergenic </span><span style='color: #BB0000;'>NA</span><span>       
#&gt; </span><span style='color: #555555;'>18</span><span> 24299759              2 intergenic </span><span style='color: #BB0000;'>NA</span><span>       
#&gt; </div><div class='input'>
<span class='co'># Get an association by association identifier</span>
<span class='fu'>get_associations</span><span class='op'>(</span>association_id <span class='op'>=</span> <span class='st'>'25389945'</span><span class='op'>)</span>
</div><div class='output co'>#&gt; An object of class "associations"
#&gt; Slot "associations":
#&gt; </span><span style='color: #555555;'># A tibble: 1 x 17</span><span>
#&gt;   association_id  pvalue pvalue_descript… pvalue_mantissa pvalue_exponent
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>            </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>                      </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span>           </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span>
#&gt; </span><span style='color: #555555;'>1</span><span> 25389945       3.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>-7</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             3              -</span><span style='color: #BB0000;'>7</span><span>
#&gt; </span><span style='color: #555555;'># … with 12 more variables: multiple_snp_haplotype </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   snp_interaction </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span style='color: #555555;'>, snp_type </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>, standard_error </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>, range </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   or_per_copy_number </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>, beta_number </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>, beta_unit </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   beta_direction </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>, beta_description </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>, last_mapping_date </span><span style='color: #555555;font-style: italic;'>&lt;dttm&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   last_update_date </span><span style='color: #555555;font-style: italic;'>&lt;dttm&gt;</span><span>
#&gt; 
#&gt; Slot "loci":
#&gt; </span><span style='color: #555555;'># A tibble: 1 x 4</span><span>
#&gt;   association_id locus_id haplotype_snp_count description     
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span>               </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>           
#&gt; </span><span style='color: #555555;'>1</span><span> 25389945              1                  27 27-SNP haplotype
#&gt; 
#&gt; Slot "risk_alleles":
#&gt; </span><span style='color: #555555;'># A tibble: 27 x 7</span><span>
#&gt;    association_id locus_id variant_id risk_allele risk_frequency genome_wide
#&gt;    </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>      </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>                </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span>      
#&gt; </span><span style='color: #555555;'> 1</span><span> 25389945              1 rs9486815  G                       </span><span style='color: #BB0000;'>NA</span><span> FALSE      
#&gt; </span><span style='color: #555555;'> 2</span><span> 25389945              1 rs4245535  G                       </span><span style='color: #BB0000;'>NA</span><span> FALSE      
#&gt; </span><span style='color: #555555;'> 3</span><span> 25389945              1 rs17069173 A                       </span><span style='color: #BB0000;'>NA</span><span> FALSE      
#&gt; </span><span style='color: #555555;'> 4</span><span> 25389945              1 rs9374021  G                       </span><span style='color: #BB0000;'>NA</span><span> FALSE      
#&gt; </span><span style='color: #555555;'> 5</span><span> 25389945              1 rs9374007  A                       </span><span style='color: #BB0000;'>NA</span><span> FALSE      
#&gt; </span><span style='color: #555555;'> 6</span><span> 25389945              1 rs9386694  A                       </span><span style='color: #BB0000;'>NA</span><span> FALSE      
#&gt; </span><span style='color: #555555;'> 7</span><span> 25389945              1 rs9374002  G                       </span><span style='color: #BB0000;'>NA</span><span> FALSE      
#&gt; </span><span style='color: #555555;'> 8</span><span> 25389945              1 rs218289   A                       </span><span style='color: #BB0000;'>NA</span><span> FALSE      
#&gt; </span><span style='color: #555555;'> 9</span><span> 25389945              1 rs1064346  A                       </span><span style='color: #BB0000;'>NA</span><span> FALSE      
#&gt; </span><span style='color: #555555;'>10</span><span> 25389945              1 rs9374013  G                       </span><span style='color: #BB0000;'>NA</span><span> FALSE      
#&gt; </span><span style='color: #555555;'># … with 17 more rows, and 1 more variable: limited_list </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span>
#&gt; 
#&gt; Slot "genes":
#&gt; </span><span style='color: #555555;'># A tibble: 1 x 3</span><span>
#&gt;   association_id locus_id gene_name
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>    
#&gt; </span><span style='color: #555555;'>1</span><span> 25389945              1 OSTM1    
#&gt; 
#&gt; Slot "ensembl_ids":
#&gt; </span><span style='color: #555555;'># A tibble: 1 x 4</span><span>
#&gt;   association_id locus_id gene_name ensembl_id     
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>     </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>          
#&gt; </span><span style='color: #555555;'>1</span><span> 25389945              1 OSTM1     ENSG00000081087
#&gt; 
#&gt; Slot "entrez_ids":
#&gt; </span><span style='color: #555555;'># A tibble: 1 x 4</span><span>
#&gt;   association_id locus_id gene_name entrez_id
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>     </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>    
#&gt; </span><span style='color: #555555;'>1</span><span> 25389945              1 OSTM1     28962    
#&gt; </div><div class='input'>
<span class='co'># Get associations by variant identifier</span>
<span class='fu'>get_associations</span><span class='op'>(</span>variant_id <span class='op'>=</span> <span class='st'>'rs3798440'</span><span class='op'>)</span>
</div><div class='output co'>#&gt; An object of class "associations"
#&gt; Slot "associations":
#&gt; </span><span style='color: #555555;'># A tibble: 1 x 17</span><span>
#&gt;   association_id   pvalue pvalue_descript… pvalue_mantissa pvalue_exponent
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>                      </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span>           </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span>
#&gt; </span><span style='color: #555555;'>1</span><span> 24299710       3.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>-10</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             3             -</span><span style='color: #BB0000;'>10</span><span>
#&gt; </span><span style='color: #555555;'># … with 12 more variables: multiple_snp_haplotype </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   snp_interaction </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span style='color: #555555;'>, snp_type </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>, standard_error </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>, range </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   or_per_copy_number </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>, beta_number </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>, beta_unit </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   beta_direction </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>, beta_description </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>, last_mapping_date </span><span style='color: #555555;font-style: italic;'>&lt;dttm&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   last_update_date </span><span style='color: #555555;font-style: italic;'>&lt;dttm&gt;</span><span>
#&gt; 
#&gt; Slot "loci":
#&gt; </span><span style='color: #555555;'># A tibble: 2 x 4</span><span>
#&gt;   association_id locus_id haplotype_snp_count description          
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span>               </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>                
#&gt; </span><span style='color: #555555;'>1</span><span> 24299710              1                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; </span><span style='color: #555555;'>2</span><span> 24299710              2                  </span><span style='color: #BB0000;'>NA</span><span> SNP x SNP interaction
#&gt; 
#&gt; Slot "risk_alleles":
#&gt; </span><span style='color: #555555;'># A tibble: 2 x 7</span><span>
#&gt;   association_id locus_id variant_id risk_allele risk_frequency genome_wide
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>      </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>                </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span>      
#&gt; </span><span style='color: #555555;'>1</span><span> 24299710              1 rs3798440  A                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'>2</span><span> 24299710              2 rs9350602  C                       </span><span style='color: #BB0000;'>NA</span><span> TRUE       
#&gt; </span><span style='color: #555555;'># … with 1 more variable: limited_list </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span>
#&gt; 
#&gt; Slot "genes":
#&gt; </span><span style='color: #555555;'># A tibble: 2 x 3</span><span>
#&gt;   association_id locus_id gene_name
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>    
#&gt; </span><span style='color: #555555;'>1</span><span> 24299710              1 MYO6     
#&gt; </span><span style='color: #555555;'>2</span><span> 24299710              2 MYO6     
#&gt; 
#&gt; Slot "ensembl_ids":
#&gt; </span><span style='color: #555555;'># A tibble: 2 x 4</span><span>
#&gt;   association_id locus_id gene_name ensembl_id     
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>     </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>          
#&gt; </span><span style='color: #555555;'>1</span><span> 24299710              1 MYO6      ENSG00000196586
#&gt; </span><span style='color: #555555;'>2</span><span> 24299710              2 MYO6      ENSG00000196586
#&gt; 
#&gt; Slot "entrez_ids":
#&gt; </span><span style='color: #555555;'># A tibble: 2 x 4</span><span>
#&gt;   association_id locus_id gene_name entrez_id
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>     </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>    
#&gt; </span><span style='color: #555555;'>1</span><span> 24299710              1 MYO6      4646     
#&gt; </span><span style='color: #555555;'>2</span><span> 24299710              2 MYO6      4646     
#&gt; </div><div class='input'>
<span class='co'># Get associations by EFO trait identifier</span>
<span class='fu'>get_associations</span><span class='op'>(</span>efo_id <span class='op'>=</span> <span class='st'>'EFO_0005537'</span><span class='op'>)</span>
</div><div class='output co'>#&gt; An object of class "associations"
#&gt; Slot "associations":
#&gt; </span><span style='color: #555555;'># A tibble: 8 x 17</span><span>
#&gt;   association_id   pvalue pvalue_descript… pvalue_mantissa pvalue_exponent
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>                      </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span>           </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span>
#&gt; </span><span style='color: #555555;'>1</span><span> 38403          4.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>- 6</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             4              -</span><span style='color: #BB0000;'>6</span><span>
#&gt; </span><span style='color: #555555;'>2</span><span> 38407          2.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>- 8</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             2              -</span><span style='color: #BB0000;'>8</span><span>
#&gt; </span><span style='color: #555555;'>3</span><span> 38404          1.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>- 7</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             1              -</span><span style='color: #BB0000;'>7</span><span>
#&gt; </span><span style='color: #555555;'>4</span><span> 38406          2.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>- 8</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             2              -</span><span style='color: #BB0000;'>8</span><span>
#&gt; </span><span style='color: #555555;'>5</span><span> 38405          9.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>- 6</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             9              -</span><span style='color: #BB0000;'>6</span><span>
#&gt; </span><span style='color: #555555;'>6</span><span> 61957837       9.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>-13</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             9             -</span><span style='color: #BB0000;'>13</span><span>
#&gt; </span><span style='color: #555555;'>7</span><span> 61957842       3.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>- 8</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             3              -</span><span style='color: #BB0000;'>8</span><span>
#&gt; </span><span style='color: #555555;'>8</span><span> 61957846       1.00</span><span style='color: #555555;'>e</span><span style='color: #BB0000;'>- 8</span><span> </span><span style='color: #BB0000;'>NA</span><span>                             1              -</span><span style='color: #BB0000;'>8</span><span>
#&gt; </span><span style='color: #555555;'># … with 12 more variables: multiple_snp_haplotype </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   snp_interaction </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span style='color: #555555;'>, snp_type </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>, standard_error </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>, range </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   or_per_copy_number </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>, beta_number </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>, beta_unit </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   beta_direction </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>, beta_description </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span style='color: #555555;'>, last_mapping_date </span><span style='color: #555555;font-style: italic;'>&lt;dttm&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   last_update_date </span><span style='color: #555555;font-style: italic;'>&lt;dttm&gt;</span><span>
#&gt; 
#&gt; Slot "loci":
#&gt; </span><span style='color: #555555;'># A tibble: 8 x 4</span><span>
#&gt;   association_id locus_id haplotype_snp_count description   
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span>               </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>         
#&gt; </span><span style='color: #555555;'>1</span><span> 38403                 1                  </span><span style='color: #BB0000;'>NA</span><span> Single variant
#&gt; </span><span style='color: #555555;'>2</span><span> 38407                 1                  </span><span style='color: #BB0000;'>NA</span><span> Single variant
#&gt; </span><span style='color: #555555;'>3</span><span> 38404                 1                  </span><span style='color: #BB0000;'>NA</span><span> Single variant
#&gt; </span><span style='color: #555555;'>4</span><span> 38406                 1                  </span><span style='color: #BB0000;'>NA</span><span> Single variant
#&gt; </span><span style='color: #555555;'>5</span><span> 38405                 1                  </span><span style='color: #BB0000;'>NA</span><span> Single variant
#&gt; </span><span style='color: #555555;'>6</span><span> 61957837              1                  </span><span style='color: #BB0000;'>NA</span><span> Single variant
#&gt; </span><span style='color: #555555;'>7</span><span> 61957842              1                  </span><span style='color: #BB0000;'>NA</span><span> Single variant
#&gt; </span><span style='color: #555555;'>8</span><span> 61957846              1                  </span><span style='color: #BB0000;'>NA</span><span> Single variant
#&gt; 
#&gt; Slot "risk_alleles":
#&gt; </span><span style='color: #555555;'># A tibble: 8 x 7</span><span>
#&gt;   association_id locus_id variant_id risk_allele risk_frequency genome_wide
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>      </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>                </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span>      
#&gt; </span><span style='color: #555555;'>1</span><span> 38403                 1 rs4245739  C                    </span><span style='color: #BB0000;'>NA</span><span>    </span><span style='color: #BB0000;'>NA</span><span>         
#&gt; </span><span style='color: #555555;'>2</span><span> 38407                 1 rs10771399 G                    </span><span style='color: #BB0000;'>NA</span><span>    </span><span style='color: #BB0000;'>NA</span><span>         
#&gt; </span><span style='color: #555555;'>3</span><span> 38404                 1 rs10069690 A                    </span><span style='color: #BB0000;'>NA</span><span>    </span><span style='color: #BB0000;'>NA</span><span>         
#&gt; </span><span style='color: #555555;'>4</span><span> 38406                 1 rs2363956  C                    </span><span style='color: #BB0000;'>NA</span><span>    </span><span style='color: #BB0000;'>NA</span><span>         
#&gt; </span><span style='color: #555555;'>5</span><span> 38405                 1 rs3757318  A                    </span><span style='color: #BB0000;'>NA</span><span>    </span><span style='color: #BB0000;'>NA</span><span>         
#&gt; </span><span style='color: #555555;'>6</span><span> 61957837              1 rs17215231 C                     0.92 FALSE      
#&gt; </span><span style='color: #555555;'>7</span><span> 61957842              1 rs2464195  G                     0.63 FALSE      
#&gt; </span><span style='color: #555555;'>8</span><span> 61957846              1 rs78378222 T                     0.99 FALSE      
#&gt; </span><span style='color: #555555;'># … with 1 more variable: limited_list </span><span style='color: #555555;font-style: italic;'>&lt;lgl&gt;</span><span>
#&gt; 
#&gt; Slot "genes":
#&gt; </span><span style='color: #555555;'># A tibble: 8 x 3</span><span>
#&gt;   association_id locus_id gene_name 
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>     
#&gt; </span><span style='color: #555555;'>1</span><span> 38403                 1 MDM4      
#&gt; </span><span style='color: #555555;'>2</span><span> 38407                 1 PTHLH     
#&gt; </span><span style='color: #555555;'>3</span><span> 38404                 1 TERT      
#&gt; </span><span style='color: #555555;'>4</span><span> 38406                 1 intergenic
#&gt; </span><span style='color: #555555;'>5</span><span> 38405                 1 ESR1      
#&gt; </span><span style='color: #555555;'>6</span><span> 61957837              1 </span><span style='color: #BB0000;'>NA</span><span>        
#&gt; </span><span style='color: #555555;'>7</span><span> 61957842              1 </span><span style='color: #BB0000;'>NA</span><span>        
#&gt; </span><span style='color: #555555;'>8</span><span> 61957846              1 </span><span style='color: #BB0000;'>NA</span><span>        
#&gt; 
#&gt; Slot "ensembl_ids":
#&gt; </span><span style='color: #555555;'># A tibble: 8 x 4</span><span>
#&gt;   association_id locus_id gene_name  ensembl_id     
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>      </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>          
#&gt; </span><span style='color: #555555;'>1</span><span> 38403                 1 MDM4       ENSG00000198625
#&gt; </span><span style='color: #555555;'>2</span><span> 38407                 1 PTHLH      ENSG00000087494
#&gt; </span><span style='color: #555555;'>3</span><span> 38404                 1 TERT       ENSG00000164362
#&gt; </span><span style='color: #555555;'>4</span><span> 38406                 1 intergenic </span><span style='color: #BB0000;'>NA</span><span>             
#&gt; </span><span style='color: #555555;'>5</span><span> 38405                 1 ESR1       ENSG00000091831
#&gt; </span><span style='color: #555555;'>6</span><span> 61957837              1 </span><span style='color: #BB0000;'>NA</span><span>         </span><span style='color: #BB0000;'>NA</span><span>             
#&gt; </span><span style='color: #555555;'>7</span><span> 61957842              1 </span><span style='color: #BB0000;'>NA</span><span>         </span><span style='color: #BB0000;'>NA</span><span>             
#&gt; </span><span style='color: #555555;'>8</span><span> 61957846              1 </span><span style='color: #BB0000;'>NA</span><span>         </span><span style='color: #BB0000;'>NA</span><span>             
#&gt; 
#&gt; Slot "entrez_ids":
#&gt; </span><span style='color: #555555;'># A tibble: 8 x 4</span><span>
#&gt;   association_id locus_id gene_name  entrez_id
#&gt;   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;int&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>      </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>    
#&gt; </span><span style='color: #555555;'>1</span><span> 38403                 1 MDM4       4194     
#&gt; </span><span style='color: #555555;'>2</span><span> 38407                 1 PTHLH      5744     
#&gt; </span><span style='color: #555555;'>3</span><span> 38404                 1 TERT       7015     
#&gt; </span><span style='color: #555555;'>4</span><span> 38406                 1 intergenic </span><span style='color: #BB0000;'>NA</span><span>       
#&gt; </span><span style='color: #555555;'>5</span><span> 38405                 1 ESR1       2099     
#&gt; </span><span style='color: #555555;'>6</span><span> 61957837              1 </span><span style='color: #BB0000;'>NA</span><span>         </span><span style='color: #BB0000;'>NA</span><span>       
#&gt; </span><span style='color: #555555;'>7</span><span> 61957842              1 </span><span style='color: #BB0000;'>NA</span><span>         </span><span style='color: #BB0000;'>NA</span><span>       
#&gt; </span><span style='color: #555555;'>8</span><span> 61957846              1 </span><span style='color: #BB0000;'>NA</span><span>         </span><span style='color: #BB0000;'>NA</span><span>       
#&gt; </div><div class='input'>
</div></span></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Ramiro Magno, Ana-Teresa Maia, Centre for Biomedical Research, University of Algarve.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


